rust-actix/swag.conf

```diff
@@ -1,109 +0,0 @@
-## Version 2024/03/06 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/nginx/site-confs/default.conf.sample
-
-# redirect all traffic to https
```

Comment:
Is this for the reverse proxy? If yes, can this be deleted?

Reply:
It was deleted already heheh